app.js

```diff
@@ -1,185 +1,249 @@
-  },
-  renderManipulation: function(){
-    var arrSum = this.randomOutputSum();
-    var x = document.getElementById('southcenter');
-    for(i=0;i<9;i++){
-      var el = document.createElement('li');
-      var updatedListItem = x.appendChild(el);
-      updatedListItem.textContent = timeLabelsArray[i] + arrSum[i];
-    };
   }
-};
-southcenter.renderManipulation();
+}
+var pikePlace = new Store ("Pike Place", 17, 88, 5.2);
+pikePlace.storeLabel();
+pikePlace.renderTable();
 
-var bellevueSquare = {
-  min: 20,
-  max: 48,
-  avg: 3.3,
-  randomOutput: function(){
-    return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
-  },
-  dailyRandomOutput: function(){
-    var cookiesArray = [];
-    for(i = 0; i < 8; i++){
-      var arrayValue = this.randomOutput();
-      cookiesArray.push(arrayValue);
-    }
-    return cookiesArray;
-  },
-  randomOutputSum: function(){
-    var arraySum = this.dailyRandomOutput();
-    var sum = 0;
-    for(i = 0; i < arraySum.length; i++){
-      sum += arraySum[i];
-    }
-    arraySum.push(sum);
-    console.log(arraySum);
-    return arraySum;
-  },
-  renderManipulation: function(){
-    var arrSum = this.randomOutputSum();
-    var x = document.getElementById('bellevue_square');
-    for(i=0;i<9;i++){
-      var el = document.createElement('li');
-      var updatedListItem = x.appendChild(el);
-      updatedListItem.textContent = timeLabelsArray[i] + arrSum[i];
-    };
-  }
-};
-bellevueSquare.renderManipulation();
+var seaTacAirport = new Store ("SeaTac Airport", 6, 24, 1.2);
+seaTacAirport.storeLabel();
+seaTacAirport.renderTable();
 
-var alki = {
-  min: 3,
-  max: 24,
-  avg: 2.6,
-  randomOutput: function(){
-    return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
-  },
-  dailyRandomOutput: function(){
-    var cookiesArray = [];
-    for(i = 0; i < 8; i++){
-      var arrayValue = this.randomOutput();
-      cookiesArray.push(arrayValue);
-    }
-    return cookiesArray;
-  },
-  randomOutputSum: function(){
-    var arraySum = this.dailyRandomOutput();
-    var sum = 0;
-    for(i = 0; i < arraySum.length; i++){
-      sum += arraySum[i];
-    }
-    arraySum.push(sum);
-    console.log(arraySum);
-    return arraySum;
-  },
-  renderManipulation: function(){
-    var arrSum = this.randomOutputSum();
-    var x = document.getElementById('alki');
-    for(i=0;i<9;i++){
-      var el = document.createElement('li');
-      var updatedListItem = x.appendChild(el);
-      updatedListItem.textContent = timeLabelsArray[i] + arrSum[i];
-    };
-  }
-};
-alki.renderManipulation();
+var southcenter = new Store ("Southcenter", 11, 38, 1.9);
+southcenter.storeLabel();
+southcenter.renderTable();
+
+var bellevueSquare = new Store ("Bellevue Square", 20, 48, 3.3);
+bellevueSquare.storeLabel();
+bellevueSquare.renderTable();
+
+var alki = new Store ("Alki", 3, 24, 2.6);
+alki.storeLabel();
+alki.renderTable();
+
+// var pikePlace = {
```

Comment:
Any reason to leave in the commented out code? Nothing wrong with wanting to hang onto it, but just curious. :)